src/AppBundle/Entity/Adherent.php

```diff
@@ -446,43 +446,46 @@ public function updateMembership(MembershipRequest $membership, PostAddress $pos
      * Joins a committee as a SUPERVISOR priviledged person.
      *
      * @param Committee $committee
+     * @param string    $subscriptionDate
```

Comment:
I would remove these annotations (only keep the description) since everything is type hinted.